metadata-service/factories/src/main/resources/application.yml

```diff
@@ -159,6 +159,11 @@ kafka:
   listener:
     concurrency: ${KAFKA_LISTENER_CONCURRENCY:1}
   bootstrapServers: ${KAFKA_BOOTSTRAP_SERVER:http://localhost:9092}
+  producer:
+    retryCount: ${KAFKA_PRODUCER_RETRY_COUNT:20}
```

Comment:
For defaults these seem a bit high. They can of course be overridden for specific use cases perhaps, I've certainly never observed dropped messages to kafka, but that might be more of a concern for Saas offerings. I am interested in other thoughts on these defaults: @pedro93 ?

retry: 3
deliveryTimeout: 15000
requestTimeout: 3000

Reply:
Have sense.